adala/utils/llm.py

```diff
@@ -0,0 +1,235 @@
+import asyncio
+import instructor
+import litellm
+import traceback
+import multiprocessing as mp
+from typing import Optional, Dict, Any, List, Type, Union
+from pydantic import BaseModel, Field
+
+instructor_client = instructor.from_litellm(litellm.completion)
+async_instructor_client = instructor.from_litellm(litellm.acompletion)
+
+
+class LLMResponse(BaseModel):
+    data: Dict = Field(default_factory=dict)
+    adala_error: bool = Field(default=False, serialization_alias='_adala_error')
```

Comment:
yes, the original reason for naming `_adala_*` is because they were columns inside a Dataframe, to avoid name collisions. Now they're properly typed so that's not an issue.

Reply:
the logic with creating dataframe is still in place, nothing changed on this way. That's why `serialization_alias` is provided

Reply:
yeah, saw the serialization alias. Figure it'd be nice to (at least internally) have a more helpful name, but I don't mind too much either way